sentry/src/main/java/io/sentry/SentryTracer.java

```diff
@@ -22,8 +20,6 @@
   private final @NotNull Span root;
   private final @NotNull List<Span> children = new CopyOnWriteArrayList<>();
   private final @NotNull IHub hub;
-  private final @NotNull Contexts contexts = new Contexts();
-  private @Nullable Request request;
```

Comment:
@maciejwalkowiak since those 2 fields should be set on the Scope, I believe nothing else should be changed, is that correct? only the removal of the fields.

Reply:
Yes, as far as I can tell these fields should be removed because they are set on the scoped and set on actual transaction in `SentryClient#applyScope`.